addons/binding/org.openhab.binding.plclogo/OSGI-INF/PLCLogoHandlerFactory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2017 by the respective copyright holders.
```

Comment:
It seems that this file has a non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.